packages/neutrino/bin/start.js

```diff
@@ -1,31 +1,57 @@
 const { start } = require('../src');
 const ora = require('ora');
+const dns = require('dns');
+const os = require('os');
+
+const platformHostName = os.hostname();
+const whenIPReady = new Promise((done, failed) => {
+  dns.lookup(platformHostName, (err, ip) => {
+    if (err) {
+      failed(err);
+    } else {
+      done(ip);
+    }
+  });
+});
+
+whenIPReady.then(ip => process.env.HOST = process.env.HOST || ip);
```

Comment:
All this "dances" with IP are necessary to show in a terminal the local machine IP when Dev Server is configured to open '0.0.0.0'. In this case to get the real `host` we have to ask it from the platform. So Webpack in most cases will open 'localhost' or '0.0.0.0' if another host is not passed in `process.env.HOST`